tooling/api/src/index.ts

```diff
@@ -13,4 +13,42 @@
  * @module
  */
 
-export * from './bundle'
+import * as app from './app'
+import * as cli from './cli'
+import * as clipboard from './clipboard'
+import * as dialog from './dialog'
+import * as event from './event'
+import * as fs from './fs'
+import * as globalShortcut from './globalShortcut'
+import * as http from './http'
+import * as notification from './notification'
+import * as path from './path'
+import * as process from './process'
+import * as shell from './shell'
+import * as tauri from './tauri'
+import * as updater from './updater'
+import * as window from './window'
+import * as os from './os'
```

Comment:
This bothers me because it's the only non-alphabetised one on the list ;)

Reply:
I will fix that once we are ready to merge this PR